blackbox/docs/appendices/release-notes/3.1.2.rst

```diff
@@ -14,11 +14,9 @@ Released on 2018/10/18.
     We recommend that you upgrade to the latest 3.0 release before moving to
     3.1.2.
 
-    You cannot perform a `rolling upgrade`_ to this version. Any upgrade to this
-    version will require a `full restart upgrade`_.
-
-    When restarting, CrateDB will migrate indexes to a newer format. Depending
-    on the amount of data, this may delay node start-up time.
+    If you want to perform a `rolling upgrade`_, your current CrateDB version
+    number must be at least :ref:`version_3.1.0`. Any upgrade from a version
```

Comment:
`3.1.1` - (see my second response) - sorry, should've  clarified that in my first comment.